[ Error ]: can't evaluate field SealKey in type core.reservedSecrets

Hi Team. I'm trying to deploy marblerun into our cluster.
## Issue description
While run marblerun according to [build.md](https://github.com/edgelesssys/marblerun/blob/master/BUILD.md#create-a-manifest), while upload manifest.json to the coordinator, get error:
```
{"status":"error","data":null,"message":"in Marble client: env variable SEAL_KEY:
 template: data:1:17: executing \"data\" at \u003c.MarbleRun.SealKey\u003e: 
can't evaluate field SealKey in type core.reservedSecrets"}
```
The coordinator output:
```
127.0.0.1 - - [05/Nov/2021:05:48:36 +0000] "POST /manifest HTTP/2.0" 400 217
```
I skim this step and run the [marbles](https://github.com/edgelesssys/marblerun/blob/master/BUILD.md#run-the-marbles) with command:
```
EDG_MARBLE_TYPE=server EDG_MARBLE_UUID_FILE=$PWD/server_uuid EDG_TEST_ADDR=localhost:8001  erthost build/marble-test-enclave.signed
```
get error:
```
[erthost] loading enclave ...
[erthost] entering enclave ...
[meshentry] invoking premain
[PreMain] 2021/11/05 05:47:56 starting PreMain
[PreMain] 2021/11/05 05:47:56 fetching env variables
[PreMain] 2021/11/05 05:47:56 loading TLS Credentials
[PreMain] 2021/11/05 05:47:56 loading UUID
[PreMain] 2021/11/05 05:47:56 found UUID: f6b2414a-ec07-47ab-a5eb-afe399dae79e
[PreMain] 2021/11/05 05:47:56 generating CSR
[PreMain] 2021/11/05 05:47:56 generating quote
[PreMain] 2021/11/05 05:47:56 activating marble of type server
panic: rpc error: code = FailedPrecondition desc = cannot accept marbles in current state

goroutine 17 [running, locked to thread]:
main.ert_meshentry_premain(0x7f1ee21e6680, 0x7f1ee21e6678)
        /home/diankun/Apps/marblerun/cmd/marble-test/mesh.go:27 +0x19d
```
The coordinator output:
```
{"level":"warn","ts":1636091252.6747434,"caller":"zap/options.go:203","msg":"finished unary call with code FailedPrecondition","grpc.start_time":"2021-11-05T05:47:32Z","system":"grpc","span.kind":"server","grpc.service":"rpc.Marble","grpc.method":"Activate","peer.address":"127.0.0.1:58298",
"error":"rpc error: code = FailedPrecondition desc = cannot accept marbles in current state","grpc.code":"FailedPrecondition","grpc.time_ms":0}
```
## Environment
* Marblerun version: latest master, commit [c270a92](https://github.com/edgelesssys/marblerun/commit/c270a92a2998a25415c56203446e0bcb6e867980)
* Edgeless RT version: latest master, commit [823c7ed](https://github.com/edgelesssys/edgelessrt/commit/823c7edbd8d5ff1d531511e2e98a3dea0dafee5c)
* Ubuntu 20.04
* With Intel Sgx

Thanks!

Thanks, @m1ghtym0 it works. But in the net step:
 ```
EDG_MARBLE_TYPE=server EDG_MARBLE_UUID_FILE=$PWD/server_uuid EDG_TEST_ADDR=localhost:8001  erthost build/marble-test-enclave.signed
```
get error:
```
[erthost] loading enclave ...
[erthost] entering enclave ...
[meshentry] invoking premain
[PreMain] 2021/11/05 06:51:26 starting PreMain
[PreMain] 2021/11/05 06:51:26 fetching env variables
[PreMain] 2021/11/05 06:51:26 loading TLS Credentials
[PreMain] 2021/11/05 06:51:26 loading UUID
[PreMain] 2021/11/05 06:51:26 found UUID: f6b2414a-ec07-47ab-a5eb-afe399dae79e
[PreMain] 2021/11/05 06:51:26 generating CSR
[PreMain] 2021/11/05 06:51:26 generating quote
[PreMain] 2021/11/05 06:51:26 activating marble of type server
panic: rpc error: code = Unauthenticated desc = invalid quote

goroutine 17 [running, locked to thread]:
main.ert_meshentry_premain(0x7fe8621e6680, 0x7fe8621e6678)
        /home/diankun/Apps/marblerun/cmd/marble-test/mesh.go:27 +0x19d
```
The coordinator output:
```
{"level":"info","ts":1636095086.6122756,"caller":"core/marbleapi.go:55","msg":"Received activation request","MarbleType":"server"}
ERROR: :OE_JSON_INFO_PARSE_ERROR [openenclave-src/common/sgx/tcbinfo.c:oe_parse_tcb_info_json:789]
ERROR: Failed to parse TCB info. OE_JSON_INFO_PARSE_ERROR (oe_result_t=OE_JSON_INFO_PARSE_ERROR) [openenclave-src/common/sgx/collateral.c:oe_validate_revocation_list:418]
ERROR: :OE_INVALID_PARAMETER [openenclave-src/enclave/crypto/mbedtls/cert.c:oe_cert_free:713]
ERROR: Failed to validate revocation info. OE_JSON_INFO_PARSE_ERROR (oe_result_t=OE_JSON_INFO_PARSE_ERROR) [openenclave-src/common/sgx/quote.c:oe_get_sgx_quote_validity:774]
ERROR: Failed to validate quote. OE_JSON_INFO_PARSE_ERROR (oe_result_t=OE_JSON_INFO_PARSE_ERROR) [openenclave-src/common/sgx/quote.c:oe_verify_quote_with_sgx_endorsements:629]
{"level":"info","ts":1636095087.5042908,"caller":"zap/options.go:203",
"msg":"finished unary call with code Unauthenticated","grpc.start_time":"2021-11-05T06:51:26Z","system":"grpc","span.kind":"server","grpc.service":"rpc.Marble","grpc.method":"Activate",
"peer.address":"127.0.0.1:58302",
"error":"rpc error: code = Unauthenticated desc = invalid quote","grpc.code":"Unauthenticated","grpc.time_ms":892.0150146484375}
```
Thanks!

--- a/BUILD.md
+++ b/BUILD.md
@@ -95,7 +95,6 @@ Here's an example that has the `SecurityVersion`, `ProductID`, and `SignerID` se
 				],
 				"Env": {
 					"ROOT_CA": "{{ pem .MarbleRun.RootCA.Cert }}",
-					"SEAL_KEY": "{{ hex .MarbleRun.SealKey }}",
 					"MARBLE_CERT": "{{ pem .MarbleRun.MarbleCert.Cert }}",
 					"MARBLE_KEY": "{{ pem .MarbleRun.MarbleCert.Private }}"
 				}
@@ -109,7 +108,6 @@ Here's an example that has the `SecurityVersion`, `ProductID`, and `SignerID` se
 				],
 				"Env": {
 					"ROOT_CA": "{{ pem .MarbleRun.RootCA.Cert }}",
-					"SEAL_KEY": "{{ hex .MarbleRun.SealKey }}",
 					"MARBLE_CERT": "{{ pem .MarbleRun.MarbleCert.Cert }}",
 					"MARBLE_KEY": "{{ pem .MarbleRun.MarbleCert.Private }}"
 				}

